src/sass/lists/_base.scss

```diff
@@ -45,3 +45,10 @@ dl dd {
   flex-direction: row;
   flex-wrap: wrap;
 }
+
+.#{$prefix}-list-reset {
```

Comment:
@levimcg Is the `.rvt-list-reset` class a replacement for `.rvt-list-plain`? They seem to fill a very similar niche.

Reply:
@scottanthonymurray Good question. We made some updates to the `rvt-list-plain` utility that include the use of flexbox to properly space list items, etc. Which make it more suited for actual lists where you just don't want any bullet styling while still maintaining consistent vertical space, etc. I added this addition "reset" utility for use cases where you might want to totally nuke list styles (including the newly-added `flex` and `gap` properties) so that you can use other utility classes to create custom styles for lists of things.

So it's not a replacement, but an alternative version of an un-styled list for different use cases like when you want to turn a list items (ul or ol) into a Rivet grid without fighting against the new `rvt-list-plain` flexbox properties.